extensions/extended-blocks/paid-blocks/README.md

```diff
@@ -0,0 +1,20 @@
+Paid Blocks
+===========
+
+This shared extension takes over providing additional behavior to a block it's considered paid.
+
+This condition can depend on a few factors: is it a simple or atomic site? have the have site a paid or free plan? However, from the perspective of paid-blocks implementation, it relies on the business-layer system.
```

Comment:
Yes, let me create a card to improve the document and remove this file from here. he documentation is poor because we haven't found a final solution for this issue.